.travis/runflake8.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+# Run flake8. The first script argument is a path to Python interpreter, the
```

Comment:
Can you please add the reason why we need this wrapper to the comment?

Reply:
Done in #147.